src/tyr/route_serializer.cc

```diff
@@ -541,8 +543,18 @@ namespace {
           maneuver_type = ramp_type(prior_edge, idx, path_leg);
         } else if (new_name) {
           maneuver_type = "new name";
-        } else if (roundabout) {
-          maneuver_type = "roundabout";
+        } else if (maneuver.type() == odin::TripDirections_Maneuver_Type_kRoundaboutEnter) {
```

Comment:
should roundabout come before new_name? maybe it should be split out like arrive and depart?

Reply:
sure